docs/getting_started/installation.md

```diff
@@ -85,6 +85,9 @@ N rows in set (0.00 sec)
 
 ### iRODS Setup
 
+!!! Important
+    iRODS 4.3.0 has a dependency on the Python 3 library, `pyodbc`. Setup will fail if this library is not installed. CentOS 7 does not provide a RPM package for this library so it isn't declared as a dependency by the iRODS packages. Therefore, administrators must manually install the package. This can be accomplished via `pip`. For example, `sudo python3 -m pip install pyodbc`.
```

Comment:
"a RPM package" -> "an RPM package"

Reply:
English is so difficult. I felt that using "an" sounded better, but I figured "a" was the correct form because ... anyway, english is hard lol.

Will update.

Reply:
done